src/api/flaskr/service/shifu/admin.py

```
@@ -6127,6 +6391,107 @@ def get_operator_user_credits(
+
+
+def get_operator_course_overview(app: Flask) -> AdminOperationCourseOverviewDTO:
+    with app.app_context():
+        return _build_operator_course_overview(app)
+
+
+def _can_use_operator_course_sql_optimization(app: Flask) -> bool:
+    try:
+        app_engines = getattr(db, "_app_engines", {})
+        return app in app_engines
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)